examples/stats/plot_sensor_regression.py

```diff
@@ -46,9 +47,25 @@
 df = pd.DataFrame(epochs.metadata)
 epochs.metadata = df.assign(Intercept=[1 for _ in epochs.events])
 
-# Run and visualize the regression
+# plot Evoked response per percentile bin of the feature
 names = ["Intercept", "Concreteness", "BigramFrequency"]
-res = linear_regression(epochs, epochs.metadata[names], names=names)
+for name in names[1:]:
+    values = epochs.metadata[name].copy()
+    values -= values.min()
+    values /= values.max()
+    values = values.round(1)
+    colors = {str(val): val for val in values}
+    df = epochs.metadata
+    name_ = name + "_percentile"
+    df[name_] = values
+    epochs.metadata = df
+    evokeds = dict()
+    for val in epochs.metadata[name_].unique():
+        evokeds[str(val)] = epochs[name_ + " == " + str(val)].average()
```

Comment:
I just feel that this code looks too ad hoc with more code to hack the metadata than doing the analysis.

maybe doing it just for Contreteness, without for loop using pd.cut can slim this down to less than 5 lines?

Reply:
Ok, I see your point. I will try!